### PR TITLE
External adapters (bridges) authenticate communication w access tokens

### PR DESCRIPTION
**Pull request by dimroc**
https://www.pivotaltracker.com/story/show/159210387

### CODE REVIEW

File: adapters/bridge.go [hunk 1]
```diff
@@ -77,13 +77,21 @@ func responseToRunResult(body []byte, input models.RunResult) models.RunResult {
...
 	if err != nil {
 		return nil, fmt.Errorf("marshaling request body: %v", err)
 	}
 
-	resp, err := http.Post(url.String(), "application/json", bytes.NewBuffer(in))
+	request, err := http.NewRequest("POST", ba.URL.String(), bytes.NewBuffer(in))
+	if err != nil {
+		return nil, fmt.Errorf("building outgoing bridge http post: %v", err)
+	}
+	request.Header.Set("Authorization", "Bearer "+ba.BridgeType.OutgoingToken)
```
Comment (thodges-gh) [inline adapters/bridge.go]: This looks kind of awkward on the external adapter receiving the outgoing token. Would you have anything against removing the "Bearer " prefix?

Comment (dimroc) [reply adapters/bridge.go]: I’m doing it to comply with the HTTP 1.0 standard. I believe it to be standard practice. For example, this differentiates is from a basic auth header which looks like:

Auhtorization: Basic someinfo

https://security.stackexchange.com/questions/108662/why-is-bearer-required-before-the-token-in-authorization-header-in-a-http-re

Comment (se3000) [inline adapters/bridge.go]: 👍 for "Bearer"

**Comment by thodges-gh** [comment]
One last thing that I found is that patching to `/v2/runs/$runId` still requires a cookie. I can open a separate bug for that if you want.

**Comment by dimroc** [comment]
Yes, please do. I'll get on it.

**Pull request merged**